File: librarymanagementsystem/src/main/java/com/yogeswari/librarymanagementsystem/dto/PendingAndPaidFinesDTO.java
package com.yogeswari.librarymanagementsystem.dto;

import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;
import java.time.LocalDate;

@Getter
@Setter
public class PendingAndPaidFinesDTO {

    private Long loanId;
    private String isbn;
    private String bookTitle;
    private BigDecimal fineAmt;
    private LocalDate dueDate;
    private LocalDate dateIn;
    private LocalDate dateOut;
    private boolean bookOut;

}

File: librarymanagementsystemui/src/app/interfaces/pending-and-paid-fines-dto.ts
export interface PendingAndPaidFinesDTO {
    
        loanId:number;      
        isbn:string;
        bookTitle:string;    
        fineAmt: number; 
        dueDate: Date;
        dateIn:Date;
        dateOut:Date;
        bookOut: boolean;    
    
}
